src/main/resources/db/changelog/changes/v1.1.0/insert-organizations-contact-info-entity-type.xml

```diff
@@ -296,15 +355,67 @@
+                      "dataType": {
+                        "dataType": "arrayType",
+                        "itemDataType": { "dataType": "stringType" }
+                      },
+                      "queryable": true,
+                      "filterValueGetter": "( SELECT array_agg(lower(cats.jsonb->>'value')) FROM jsonb_array_elements(org.jsonb->'urls') as url, jsonb_array_elements(url->'categories') as category_id JOIN src_organization_categories cats ON cats.id = (category_id#>>'{}')::uuid )"
                     }
                   ]
                 }
               },
-              "valueGetter": "org.jsonb ->> 'urls'",
+              "valueGetter": "( SELECT jsonb_agg(url || jsonb_build_object('categoryNames', COALESCE(( SELECT array_agg(cats.jsonb->>'value') FROM jsonb_array_elements(url->'categories') as category_id JOIN src_organization_categories cats on cats.id = (category_id#>>'{}')::uuid ), array []::text[])))::text FROM jsonb_array_elements(org.jsonb->'urls') as url)",
+              "visibleByDefault": false
+            },
+            {
+              "name": "acquisition_unit_id",
+              "dataType": {
+                "dataType": "arrayType",
+                "itemDataType": {
+                  "dataType": "rangedUUIDType"
+                }
+              },
+              "queryable": false,
+              "valueGetter": "( SELECT array_agg(acq_id.value::text) FILTER (WHERE (acq_id.value::text) IS NOT NULL) AS array_agg FROM jsonb_array_elements_text(org.jsonb -> 'acqUnitIds'::text) acq_id(value))",
+              "filterValueGetter": "( SELECT array_agg(lower(acq_id.value::text)) FILTER (WHERE (acq_id.value::text) IS NOT NULL) AS array_agg FROM jsonb_array_elements_text(org.jsonb -> 'acqUnitIds'::text) acq_id(value))",
+              "valueFunction": "lower(:value)",
               "visibleByDefault": false
+            },
+            {
+              "name": "acquisition_unit_name",
+              "dataType": {
+                "dataType": "arrayType",
+                "itemDataType": {
+                  "dataType": "stringType"
+                }
+              },
+              "queryable": true,
+              "valueGetter": "( SELECT array_agg(acq_unit.jsonb ->> 'name'::text) FILTER (WHERE (acq_unit.jsonb ->> 'name'::text) IS NOT NULL) AS array_agg FROM jsonb_array_elements_text((org.jsonb -> 'acqUnitIds'::text)) record(value) JOIN src_acquisitions_unit acq_unit ON lower(record.value::text) = acq_unit.id::text)",
+              "filterValueGetter": "( SELECT array_agg(lower(acq_unit.jsonb ->> 'name'::text)) FILTER (WHERE (acq_unit.jsonb ->> 'name'::text) IS NOT NULL) AS array_agg FROM jsonb_array_elements_text((org.jsonb -> 'acqUnitIds'::text)) record(value) JOIN src_acquisitions_unit acq_unit ON (record.value::text) = acq_unit.id::text)",
+              "valueFunction": "lower(:value)",
+              "visibleByDefault": false,
+              "idColumnName": "acquisition_unit_name",
```

Comment:
should this be acquisition_unit_id?

Reply:
it should be` "idColumnName": "acqunit_ids"`